extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/client/OidcClientConfig.java

```diff
@@ -71,7 +71,11 @@ public static enum Type {
              * at least 'subject_token' parameter which must be passed to OidcClient at the token request time.
              */
             EXCHANGE("urn:ietf:params:oauth:grant-type:token-exchange"),
-
+            /**
+             * 'urn:ietf:params:oauth:grant-type:jwt-bearer' grant requiring an OIDC client authentication as well as
+             * at least an 'assertion' parameter which must be passed to OidcClient at the token request time.
+             */
+            JWTBEARER("urn:ietf:params:oauth:grant-type:jwt-bearer"),
```

Comment:
```suggestion
            JWT_BEARER("urn:ietf:params:oauth:grant-type:jwt-bearer"),
```

Reply:
Hi @gastaldi One thing I've been following with these grant names as introducing the shortcuts avoiding `-`, `_`, for example, `client_credentials` -> `client`, `urn:ietf:params:oauth:grant-type:token-exchange` -> `exchange`, etc. One thing it avoids is hard to catch issues with the extra parameters which I vaguely recall we had when `client` was named as `client_credentials`, where the user typed `extra-params.client-credentials.a=b`.
I've been thinking  though earlier to have even a shorter shortcut, `jwt` -> `urn:ietf:params:oauth:grant-type:jwt-bearer` - perhaps we can try this one ?

Reply:
That makes sense, yes

Reply:
Sorry for keeping pushing, just noticed it :-)

Reply:
Done, `jwt` is much simpler, thanks for suggesting a grant name change in the first place as well